src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js

```diff
@@ -125,7 +125,12 @@ const VIEWPORT_TOP_OFFSET_DISTRACTION_FREE_MODE = 0;
             const wrapper = this.getHTMLDocumentFragment(container.closest('.ibexa-data-source').querySelector('textarea').value);
             const section = wrapper.childNodes[0];
             const { toolbar, extraPlugins = [], extraConfig = {} } = window.ibexa.richText.CKEditor;
-            const locale = new Intl.Locale(doc.querySelector('meta[name="LanguageCode"]').content);
+            let locale;
+            try {
+                locale = new Intl.Locale(doc.querySelector('meta[name="LanguageCode"]').content);
+            } catch (e) {
+                locale = new Intl.Locale('eng-GB');
```

Comment:
I would add some kind of `console.warn` to catch to give user any kind of message, that something has gone wrong - otherwise it's silent error and user having lang `ger` can be suprised why it's not working correctly and uses eng locale

Reply:
I have added `console.warn` now.